content/en/blog/2022/k8s-otel-expose.md

```diff
@@ -0,0 +1,404 @@
+---
+title: Exposing an OpenTelemetry Collector for cross cluster communication
+linkTitle: Exposing an OpenTelemetry Collector
+date: 2022-08-25
+author: '[Benedikt Bongartz](https://github.com/frzifus)'
+spelling:
+  cSpell:ignore k8sattributes k8sattributesprocessor K8sattributes k8sprocessor
+  cSpell:ignore K8sprocessor KUBE
+---
+
+Exposing an [OpenTelemetry Collector](/docs/collector/) currently requires
+a number of configuration steps. The goal of this blog post is to demonstrate
+`how to establish a secure communication` between two collectors in different
+kubernetes clusters.
+
+
+Details of CRDs and dependency installations are not covered by this post.
+
+
+# Overview
+
+When it comes to making collectors publicly accessible, the first thing that
+comes to mind is the secure transmission of user data via TLS. However,
+authentication to the server is at least as important to prevent unauthorized
+services from sending data.
+
+The OpenTelemetry Collector supports different authentication methods. The
+most used are probably:
+1. TLS Authentification
+2. OpenID Connect (OIDC-Authentication)
+3. HTTP Basic Authentication
+
+This article focuses on `HTTP Basic Authentication` for simplicity. It is intended to show how
+a secure setup can be operated without key management or further thrid party services.
+
+
+For more information about TLS configuration I would like to refer to the article
+[How TLS provides identification, authentication, confidentiality, and integrity](https://www.ibm.com/docs/en/ibm-mq/9.1?topic=tls-how-provides-identification-authentication-confidentiality-integrity)
+and the Collector [TLS-Config](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.58.0/config/configtls/README.md)
+description on Github.
+
+
+If you are interested in using an external authentication provider, I advise you to
+have a look at the article `Securing your OpenTelemetry Collector` of Juraci Paixão
+Kröhling on this topic. He explains how OpenTelemetry collectors can be secured using the
+[OIDC-Authenticator extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.58.0/extension/oidcauthextension),
+and how [Keycloak](https://www.keycloak.org/) can be configured as an authentication provider.
+
+
+# Basic Authentication
+
+The HTTP Basic Authentication mechanism is quite simple. An HTTP user agent
+(e.g., a web browser) provides a username and password combination on every request.
+Transmitted credentials are included in the HTTP header by the key `Authorization`
+when the connection is established. As value the authentication method `basic` is
```

Comment:
As a value OR you could just leave that out and start the sentence with "The authentication method..."